spec/suites/map/MapSpec.js

```diff
@@ -1336,4 +1336,57 @@ describe("Map", function () {
 			}).to.not.throwException();
 		});
 	});
+
+	describe("_addZoomLimit", function () {
+		it("update zoom levels when min zoom is not NaN in a layer that is added to map", function () {
+			map._addZoomLimit(L.tileLayer("", {minZoom: 4}));
+			expect(map._layersMinZoom).to.be(4);
+		});
+
+		it("update zoom levels when max zoom is not NaN in a layer that is added to map", function () {
+			map._addZoomLimit(L.tileLayer("", {maxZoom: 10}));
+			expect(map._layersMaxZoom).to.be(10);
+		});
+
+		it("update zoom levels when min zoom is not NaN in two layers that are added to map", function () {
+			map._addZoomLimit(L.tileLayer("", {minZoom: 6}));
+			map._addZoomLimit(L.tileLayer("", {minZoom: 4}));
+			expect(map._layersMinZoom).to.be(4);
+		});
+
+		it("update zoom levels when max zoom is not NaN in two layers that are added to map", function () {
+			map._addZoomLimit(L.tileLayer("", {maxZoom: 10}));
+			map._addZoomLimit(L.tileLayer("", {maxZoom: 8}));
+			expect(map._layersMaxZoom).to.be(10);
+		});
+
+		it("update zoom levels, with min zoom level to NaN, when min zoom is NaN in a layer that is added to map", function () {
```

Comment:
Please add to each `NaN` test following line:
> // This test shows the NaN usage - it's not clear if NaN is a wanted "feature"

I think we should add this comment because else in the future we maybe think `NaN` is a wanted feature.

Reply:
Added comments on top of each test function that shows NaN usage for min zoom or max zoom.